sklearn/feature_selection/sequential_feature_selector.py

```diff
@@ -0,0 +1,332 @@
+
+    n_features_to_select : int or tuple (default=1)
+        An integer arguments specifies the number of features to select,
+        where n_features_to_select < the full feature set.
+        Optionally, a tuple containing a min and max value can be provided
+        so that the feature selector will return a feature subset between
+        with min <= n_features <= max that scored highest in the evaluation.
+        For example, the tuple (1, 4) will return any combination from
+        1 up to 4 features instead of a fixed number
+        of features `n_features_to_select`.
+
+    scoring : str, callable, or None (default=None)
+        A string (see model evaluation documentation) or a scorer
+        callable object / function with signature `scorer(estimator, X, y)`.
+
+    forward : bool (default=True)
+        Performs forward selection if True and backward selection, otherwise.
+
+    cv : int or cross-validation generator, or an iterable (default=5)
+        Determines the cross-validation splitting strategy for
+        feature selection. Possible inputs for cv are:
+        - 0 or None, don't use cross validation
+        - integer > 1, to specify the number of folds in a (Stratified)KFold
+        - An object to be used as a cross-validation generator.
+        - An iterable yielding train, test splits.
+        For integer/None inputs, if the estimator is a classifier
+        and `y` is either binary or multiclass, `StratifiedKFold` is used.
+        In all other cases, `KFold` is used.
+
+    n_jobs : int (default=1)
+        The number of CPUs to use for cross validation.
+
+    pre_dispatch : int, or string (default: '2*n_jobs')
+        Controls the number of jobs that get dispatched
+        during parallel execution in cross_val_score.
+        Reducing this number can be useful to avoid an explosion of
+        memory consumption when more jobs get dispatched than CPUs can process.
+        This parameter can be:
+        - None, in which case all the jobs are immediately created and spawned.
+          Use this for lightweight and fast-running jobs,
+          to avoid delays due to on-demand spawning of the jobs
+        - An int, giving the exact number of total jobs that are spawned
+        - A string, giving an expression as a function
+            of `n_jobs`, as in `2*n_jobs`
+
+    Attributes
+    ----------
+    support_ : array of shape [n_features]
+        The mask of selected features.
+
+    score_ : float
+        Cross validation average score of the selected subset.
+
+    subsets_ : dict
+        A dictionary containing the best selected feature subset
+        for each feature subset size selected by the
+        sequential feature selection algorithm.
+        Here the dictionary keys are the lengths k of these feature subsets.
+        The dictionary values are dictionaries themselves with the following
+        keys: 'feature_subset_idx' (tuple of indices of the feature subset)
```

Comment:
I think this can also be called 'support' or 'support_idx'.

The formatting here won't work in HTML. Perhaps use ReST definition lists.

Reply:
done